MHVLib/Types/MHVPlanObjective.h

```diff
@@ -26,10 +26,16 @@
 @property (readwrite, nonatomic, strong) MHVStringNZNW *name;
 @property (readwrite, nonatomic, strong) MHVStringNZNW *descriptionText;
 @property (readwrite, nonatomic, strong) NSString *state;
-@property (readwrite, nonatomic, strong) MHVPlanOutcomeCollection *outcomes;
+@property (readwrite, nonatomic, strong) MHVPlanOutcomeList *outcomes;
```

Comment:
I believe this is the object used for actions plans, but it is not called such in the HV docs. Even if we support it with the REST services, we should have it in the core HV code I thought?